lib/stats/stats-registry.c

```diff
@@ -355,6 +355,15 @@ stats_get_cluster(const StatsClusterKey *sc_key)
   return sc;
 }
 
+gboolean
+stats_remove_cluster(const StatsClusterKey *sc_key)
+{
+  g_assert(stats_locked);
+
+  return g_hash_table_remove(stats_cluster_container.dynamic_clusters, sc_key) ||
+         g_hash_table_remove(stats_cluster_container.static_clusters, sc_key);
+}
+
```

Comment:
@MrAnno Should the stats API only allow removing if the cluster is abandoned?

Reply:
Orphaned* ...

Reply:
I will fix this. Is it okay to do in a separate PR, if you find this one good except this?

Reply:
As you see fit.

Reply:
Fixed.